.github/workflows/model_uploader.yml

```diff
@@ -96,16 +110,35 @@ jobs:
         echo "${workflow_info@E}" >> $GITHUB_OUTPUT
         echo "EOF" >> $GITHUB_OUTPUT
         echo "${workflow_info@E}"
+    - name: Check MIT license url
+      id: check_mit_license_url
+      run: |
+        if [[ "${{ github.event.inputs.model_license }}" == "MIT" ]]
+        then
+          echo "Uploading MIT licensed model"
+          if [[ "${{ github.event.inputs.MIT_license_url }}" == "" ]]
+          then
+              echo "missing MIT license url"
+              exit 1
+          fi
+          if [[ "${{ github.event.inputs.model_source }}" == "" ]]
```

Comment:
what if we provide some other source?

Reply:
Then the third party statements can be wrong. So we need to check whether they are matched before approve the release

Reply:
What I tried to mean is let's say `github.event.inputs.model_source` == "XYZ" then this condition will pass, right?